CesiumGeometry/test/TestIntersectionTests.cpp

```diff
@@ -38,6 +38,96 @@ TEST_CASE("IntersectionTests::rayPlane") {
   CHECK(intersectionPoint == testCase.expectedIntersectionPoint);
 }
 
+const glm::dvec3 unitInverseRadii(1.0, 1.0, 1.0);
+const glm::dvec3 wgs84Radii(6378137.0, 6378137.0, 6356752.3142451793);
+const glm::dvec3 wgs84InverseRadii = 1.0 / wgs84Radii;
```

Comment:
```suggestion
namespace {
  const glm::dvec3 unitInverseRadii(1.0, 1.0, 1.0);
  const glm::dvec3 wgs84Radii(6378137.0, 6378137.0, 6356752.3142451793);
  const glm::dvec3 wgs84InverseRadii = 1.0 / wgs84Radii;
}
```

Reply:
I think it would be fine to `#include <CesiumGeospatial/Ellipsoid.h>` in this file as well, even though it's testing `CesiumGeometry` functionality.

Reply:
already done.